float64 joy_time
float32 joy_speed
float32 joy_dir
float32 joy_yaw
bool autonomy_mode
bool check_obstacle
